models/base/course_names.sql

```diff
@@ -0,0 +1,28 @@
+{{
+    config(
+        materialized="dictionary",
+        schema=env_var("ASPECTS_EVENT_SINK_DATABASE", "event_sink"),
+        fields=[
+            ("course_key", "String"),
+            ("course_name", "String"),
+        ],
+        primary_key="course_key",
+        layout="COMPLEX_KEY_HASHED()",
+        lifetime="120",
+        source_type="clickhouse",
+    )
+}}
+
+with
+    most_recent_overviews as (
+        select org, course_key, max(modified) as last_modified
+        from {{ ASPECTS_EVENT_SINK_DATABASE }}.course_overviews
```

Comment:
This could be updated to `source("event_sink", "course_overviews")` as well, like below.